<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="restAPITestCases01.RevenueBundle" tests="4" time="5.387" ignored="0" failures="2" timestamp="16 Jan 2018 06:01:04 GMT" skipped="0" hostname="BAN-PATILSUW10" errors="0">
  <testcase name="gettingRevenueBundles" time="2.133" classname="restAPITestCases01.RevenueBundle"/>
  <testcase name="obl_getObligations01" time="1.467" classname="restAPITestCases01.RevenueBundle">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
JSON path Obligations.ObligationID doesn&#039;t match.
Expected: (a collection containing &lt;147&gt; and a collection containing &lt;148&gt; and a collection containing &lt;149&gt;)
  Actual: [47, 48, 49, 50, 51, 52, 57, 66, 67, 68, 52, 64, 65, 128, 129, 1833, 75, 1805, 53, 54, 55, 56, 57, 81, 81, 82, 82, 84, 84, 85, 85, 57]
 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path Obligations.ObligationID doesn't match.
Expected: (a collection containing <147> and a collection containing <148> and a collection containing <149>)
  Actual: [47, 48, 49, 50, 51, 52, 57, 66, 67, 68, 52, 64, 65, 128, 129, 1833, 75, 1805, 53, 54, 55, 56, 57, 81, 81, 82, 82, 84, 84, 85, 85, 57]
 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at restAPITestCases01.RevenueBundle.obl_getObligations01(RevenueBundle.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- obl_getObligations01 -->
  <testcase name="obl_getObligations" time="0.611" classname="restAPITestCases01.RevenueBundle">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
JSON path Obligations.ObligationID doesn&#039;t match.
Expected: (a collection containing &lt;47&gt; and a collection containing &lt;48&gt; and a collection containing &lt;49&gt;)
  Actual: null
 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path Obligations.ObligationID doesn't match.
Expected: (a collection containing <47> and a collection containing <48> and a collection containing <49>)
  Actual: null
 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at restAPITestCases01.RevenueBundle.obl_getObligations(RevenueBundle.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- obl_getObligations -->
  <testcase name="gettingRevenueBundles02" time="1.176" classname="restAPITestCases01.RevenueBundle"/>
</testsuite> <!-- restAPITestCases01.RevenueBundle -->
